src/gettingstarted/cli/api-tokens.md

```diff
@@ -0,0 +1,63 @@
+# API Tokens
+
+## Obtaining a token
+
+The Platform.sh CLI can also be used from CI services or other automation tools, and supports an API Token authentication option for that purpose.
+
+An API token can be created through the UI.  Go to the "User" page from your accounts drop-down, then select the "Account Settings" tab, then the "API Tokens".
+
+Click the "Create an API Token" link.  You may be asked to reverify your password, then enter a unique application name to identify the token.
+
+![API Token screen](/images/api-token-list.png)
```

Comment:
I believe this needs to be `src/images/api-token-list.png`

Looking at the preview: https://github.com/platformsh/platformsh-docs/blob/cli-backups/src/gettingstarted/cli/api-tokens.md

You can see the image does not display.

Reply:
It's correct - have a look at https://pr-718-7beqdoa-t2llqeifuhpzg.eu.platform.sh/gettingstarted/cli/api-tokens.html